public/components/overview/github-panel/github-panel.tsx

```diff
@@ -11,18 +11,23 @@
  * Find more information about this on the LICENSE file.
  */
 
-import React from 'react';
+import React, {useState} from 'react';
 import { MainPanel } from '../../common/modules/panel';
 import { withErrorBoundary } from '../../common/hocs';
 import { CustomSearchBar } from '../../common/custom-search-bar';
 import { ModuleConfiguration } from './views';
 import { ModuleConfig, filtersValues } from './config';
 
 export const GitHubPanel = withErrorBoundary(() => {
+  const [drillDownValue, setDrillDownValue] = useState({ field: '', value: '' });
+  const filterDrillDownValue = (value) => {
+    setDrillDownValue(value)
+  }
```

Comment:
Any reason why the `filterDrillDownValue` should only filter by the filter selected when drilling down instead of the global filter state?

Reply:
If we use global filter, we can have some problems distinguishing which filter we can pick to the boolFilter. I think that catch them when you click will be a better option.